eclair-web/crate/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "eclair-wasm"
+version = "0.1.0"
+authors = ["Yan <mindv0rtex@users.noreply.github.com>"]
+edition = "2018"
+
+[lib]
+crate-type = ["cdylib", "rlib"]
```

Comment:
Curious why we need both types here.

Reply:
`"cdylib"` is necessary for making WASM libs, `"rlib"` is helpful for testing -- TIL, see https://rustwasm.github.io/wasm-pack/book/tutorials/npm-browser-packages/template-deep-dive/cargo-toml.html#1-crate-type.